webpack/stories/index.js

```diff
@@ -112,3 +114,29 @@ storiesOf('Power Status', module)
       />
     )
   );
+
+function dismiss() {
+  console.log('callback executed');
```

Comment:
You might want to take a look at the [patternfly-react](https://jgiardino.github.io/patternfly-react/?knob-Label=Danger%20Will%20Robinson%21&selectedKind=Alert&selectedStory=with%20danger&full=0&down=1&left=1&panelRight=0&downPanel=kadirahq%2Fstorybook-addon-actions%2Factions-panel) implementation using an action logger